src/Pages/VolunteerPage.jsx

```diff
@@ -89,59 +126,155 @@ const VolunteerPage = () => {
-                </li>
-                <li>
-                  You are self-motivated and are comfortable with remote work
-                </li>
-              </BulletedListMui>
-            </Box>
+              <List>
+                <ListItem>
+                  <ListItemIcon>
+                    <Alarm className={classes.icon} />
+                  </ListItemIcon>
+                  <Typography className={classes.listTextAlt}>
+                    You can commit to 4 hours a week for at least 6 months
+                  </Typography>
+                </ListItem>
+                <ListItem>
+                  <ListItemIcon>
+                    <SchoolOutlined className={classes.icon} />
+                  </ListItemIcon>
+                  <Typography className={classes.listTextAlt}>
+                    You want to share your expertise in a field to help us make
+                    a better product
+                  </Typography>
+                </ListItem>
+                <ListItem>
+                  <ListItemIcon>
+                    <CloudDoneOutlined className={classes.icon} />
+                  </ListItemIcon>
+                  <Typography className={classes.listTextAlt}>
+                    You are self-motivated and are comfortable with remote work
+                  </Typography>
+                </ListItem>
+              </List>
+            </Grid>
+            <Grid
+              className={classes.backgroundSecondary}
+              item
+              xs={12}
+              sm={6}
+              md={6}
+            >
+              <Box>
+                <Typography
+                  className={classes.titleAlt}
+                  variant="h2"
+                  align="right"
+                >
+                  How can you apply?
+                </Typography>
+                <Typography
+                  className={classes.contentTextAlt}
+                  variant="body1"
+                  align="right"
+                >
+                  We are currently using DemocracyLab to streamline the
+                  volunteering process, which you can find by clicking the
+                  button below.
+                </Typography>
+                <Typography
+                  className={classes.contentTextAlt}
+                  variant="body1"
+                  align="right"
+                >
+                  If you want further information and want to have a chat, don’t
+                  hesitate to contact us by email at{" "}
+                  <a href="/contact">contact@cvp.com</a> and we’ll get back to
+                  you within 24 hours.
+                </Typography>
+              </Box>
+            </Grid>
           </Grid>
-          <Grid item md={6} className={classes.backgroundWhite}>
-            <Box className={classes.contentBox}>
-              <Typography className={classes.textAltColor} variant="h2">
-                How can you apply?
+
+        <Box>
```

Comment:
Can we have these images and headers aligned horizontally? 

<img width="1257" alt="Screen Shot 2022-01-17 at 10 05 45 AM" src="https://user-images.githubusercontent.com/25576005/149819348-c744d8d8-7fd7-4f73-b047-6ec81c40f35a.png">

Reply:
Yes I'll take a look. It might need asymmetric padding.

Reply:
Ah I see. I was matching the design in [figma](https://www.figma.com/file/icusYj7kJl2xmYkudIZEG4/CVP-Design-Files?node-id=4210%3A7776). I did that on purpose. Would it be better if it was more pronounced? or you think flatten them?